client/App.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import CheckoutCalendar from './components/CheckoutCalendar/CheckoutCalendar.js';
+
+import { DisplayCalendar } from './components/DisplayCalendar/DisplayCalendar';
```

Comment:
wonder if you need to use destructuring here. im not as familiar with import/export vs module.exports/require. do you plan on exporting multiple components from one file?

Reply:
I actually thought the same but it doesn't work if I don't use destructuring here because of the way I set up the individual components I think.